### ISSUE TITLE
'NoneType' object has no attribute 'find'

### ISSUE THREAD

**Issue by tanner02** [BaraniARR/anikimiapi#11]
There seems to be an error in the API

`Traceback (most recent call last):
  File "/Users/user/Library/Python/3.9/lib/python/site-packages/anikimiapi/anikimi.py", line 218, in get_episode_link_advanced
    source_url = lnk.find("li").a
AttributeError: 'NoneType' object has no attribute 'find'`

**Comment by ryan-k8** [comment]
yes, this was simply because the host website has been updated and gives 403 error code if user-agent is not set in request headers. i will make a issue and consequently a PR for fixing this.

**Comment by tanner02** [comment]
Thank you, I'm looking forward to it!
